pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -577,6 +571,15 @@ def is_subrepo(directory):
         with RpmPublication.create(
             repo_sync_results[PRIMARY_REPO], pass_through=False
         ) as publication:
+            gpgcheck = repository.repo_config.get("gpgcheck", 0)
+            has_repomd_signature = (
```

Comment:
I wonder if instead of looking at .asc file we should be fetching the repo_gpgcheck from repository.repo_config?

Reply:
How about: if it's enabled and the .asc exists then we enable it, if either is false then we disable it.

Reply:
yes, works for me